Assets/Scripts/Game/DaggerfallUI.cs

```diff
@@ -43,9 +43,10 @@ public class DaggerfallUI : MonoBehaviour
         public static Color DaggerfallDefaultShadowColor = new Color32(93, 77, 12, 255);
         public static Color DaggerfallAlternateShadowColor1 = new Color32(44, 60, 60, 255);
         public static Color DaggerfallDefaultSelectedTextColor = new Color32(162, 36, 12, 255);
+        public static Color DaggerfallBrighterSelectedTextColor = new Color32(254, 56, 18, 255);
         public static Color DaggerfallUnityStatDrainedTextColor = new Color32(190, 85, 24, 255);
         public static Color DaggerfallUnityStatIncreasedTextColor = new Color32(178, 207, 255, 255);
-        public static Color DaggerfallDefaultTextCursorColor = new Color32(154, 134, 0, 200);
```

Comment:
I just realised you overwrote the text cursor colour here. This is the blinking I-beam in text entry, and it shouldn't be this color. I'll reset this one now.

![pink-ibeam](https://user-images.githubusercontent.com/10426244/46450550-26efa080-c7d5-11e8-89bc-12949a4dac3d.png)